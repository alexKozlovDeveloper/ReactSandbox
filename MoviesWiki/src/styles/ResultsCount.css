.container {
    font-family: cursive;
    float: left;
    color: white;
    margin-left: 15px;
    margin-bottom: 5px;
    margin-top: 14px;
}