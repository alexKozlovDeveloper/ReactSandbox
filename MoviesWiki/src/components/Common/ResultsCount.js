import React, { Component } from "react";

import styles from "../../styles/ResultsCount.css"

const ResultsCount = ({ count, title }) => {
    return <div className={styles.container}>
        {count} {title}
    </div>;
}

export default ResultsCount;