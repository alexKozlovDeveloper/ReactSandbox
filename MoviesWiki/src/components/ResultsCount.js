import React, { Component } from "react";

import styles from "../styles/ResultsCount.css"

function ResultsCount(props) {
    return  <div className={styles.container}>
                {props.count} {props.title}
            </div>;
}

export default ResultsCount;