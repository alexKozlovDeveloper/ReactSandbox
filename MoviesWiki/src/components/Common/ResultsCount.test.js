import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ResultsCount from "./ResultsCount";

configure({adapter: new Adapter()});

test('ResultsCount rendering test', () => {

    var count = "someCount";
    var title = "someTitle";

    const component = shallow(<ResultsCount count={count} title={title}/>);
    expect(shallowToJson(component)).toMatchSnapshot();
 });

