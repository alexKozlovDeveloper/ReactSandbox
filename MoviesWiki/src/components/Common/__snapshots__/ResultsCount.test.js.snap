// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsCount rendering test 1`] = `
<div
  className="container"
>
  someCount
   
  someTitle
</div>
`;
